### ISSUE TITLE
 How to perform finetuning on a network which has batch normalization layer? 

### ISSUE THREAD

**Issue by wjf199105** [BVLC/caffe#4889]
How to perform Finetuning on a network which has batch normalization layer?

**Comment by williford** [comment]
Questions like these should be asked on the [Caffe user mailing list](https://groups.google.com/forum/#!forum/caffe-users) or on another website, such as StackOverflow. When you do, you should give more details about what you are having difficulties with.
